runtime/src/proposals_configuration/defaults.rs

```diff
@@ -343,3 +343,17 @@ pub(crate) fn update_channel_payouts_proposal() -> ProposalParameters<BlockNumbe
         constitutionality: 1,
     }
 }
+
+// Proposal parameters for the 'Freeze Pallet' proposal
+pub(crate) fn freeze_pallet_proposal() -> ProposalParameters<BlockNumber, Balance> {
+    ProposalParameters {
+        voting_period: days!(3),
+        grace_period: hours!(2),
```

Comment:
```suggestion
        grace_period: 0,
```

I think it was suggested by tomato @mochet that it doesn't need a grace period.

Reply:
Tomato perhaps you could also weigh in on other parameters for this proposal.

Reply:
> Tomato perhaps you could also weigh in on other parameters for this proposal.

LGTM! It has 2/3 approval vote requirement which seems enough. I don't think this kind of proposal would need 3/3 votes as it can sometimes take time to get the 3rd vote.